release/action.yaml

```diff
@@ -36,7 +36,8 @@ runs:
       uses: actions/checkout@v3
       if: inputs.checkout-repo
       with:
-        fetch-depth: 0
+        fetch-depth: ${{ inputs.checkout-repo == 'true' && 0 || 1 }}
+        persist-credentials: false
```

Comment:
Interesting @tagoro9 , is this something we should be doing across all of the `open-turo` repos?

Reply:
I'm testing this after we merge it, so it may need to be needed (if we want to allow semantic release to commit / push using the passed Github token)